programs/ziti-edge-tunnel/package/deb/postinst.in

```diff
@@ -30,10 +30,22 @@ if [ "$1" = "configure" ]; then
   chmod 0770 "@ZITI_IDENTITY_DIR@"
   find "@ZITI_IDENTITY_DIR@" -maxdepth 1 -name "*.json" -type f -exec chown ziti:ziti "{}" + -exec chmod 0400 "{}" +
 
-  # install PolicyKit policy if directory exists
-  if [ -d /var/lib/polkit-1/localauthority/10-vendor.d ]; then
-    cp "@CPACK_SHARE_DIR@/@ZITI_POLKIT_PKLA_FILE@.sample" "/var/lib/polkit-1/localauthority/10-vendor.d/@ZITI_POLKIT_PKLA_FILE@"
-    db_set ziti_edge_tunnel/install_pkla true
+  policykit_version=$(dpkg-query -Wf '${Version;5}' policykit-1 | cut -d . -f 2)
+  systemd_version=$(dpkg-query -Wf '${Version;3}' systemd)
+
+  # install PolicyKit policy if using policykit < 0.106 (https://askubuntu.com/questions/1287924/whats-going-on-with-policykit)
+  if [ ${policykit_version} -lt 106 ]; then
+    # ... the set-llmnr action was exposed with v243 (https://github.com/systemd/systemd/commit/52aaef0f5dc81b9a08d720f551eac53ac88aa596)
+    if [ ${systemd_version} -ge 243 ]; then
+      cp "@CPACK_SHARE_DIR@/@ZITI_POLKIT_PKLA_FILE@.sample" "/var/lib/polkit-1/localauthority/10-vendor.d/@ZITI_POLKIT_PKLA_FILE@"
+      db_set ziti_edge_tunnel/install_pkla true
+    else
+      service_user=root
+      override_dir="${SYSTEMD_UNIT_DIR}/${SYSTEMD_UNIT_FILE_NAME}.d"
```

Comment:
I think this needs to be `@SYSTEMD_UNIT_DIR@` and `@SYSTEMD_UNIT_FILE_NAME@` since these are cmake vars.